src/eventPage/EventList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Event from './Event'
+import styled from 'styled-components/macro'
+//import {getEvents} from '../utils/services'
+const jsonEvents = require('../mockDB/events.json')
```

Comment:
use ES6 feature "import" instead of "require"

Reply:
changed